### DISCUSSION TITLE
cron: Used memory grew by more than 10 MB

### DISCUSSION THREAD

**Discussion by Martinligabue** [nextcloud/all-in-one#4838] [Questions] [answered]
After the last update (9.0.1) I get this log from cron twice every half hour: Used memory grew by more than 10 MB when executing job OCA\Memories\Cron\IndexJob (id: 152, arguments: null): 35.8 MB (before: 25.3 MB).
The values change slightly from time to time.
I don't see any problems with the server, but it's a warning, and I haven't found any mention of it in the issues and discussions.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/server/issues/45682

**Reply by ShapeShifter499** [reply]
I have this warning here 

```
{"reqId":"u4KzAcGtZhffLCgWOQxU","level":2,"time":"2024-12-08T02:19:16+00:00","remoteAddr":"","user":"--","app":"cron","method":"","url":"--","message":"Cron job used more than 300 MB of ram after executing job OCA\\Memories\\Cron\\IndexJob (id: 134, arguments: null): 499 MB (before: 35.2 MB)","userAgent":"--","version":"30.0.2.2","data":{"app":"cron"},"id":"6755099a6bab8"}
```
